src/app/content/practiceQuestions/components/Answer/index.tsx

```diff
@@ -63,19 +72,22 @@ const Answer = ({
       id='i18n:practice-questions:popup:answers:choice'
       values={{choiceIndicator: choiceIndicator.toUpperCase()}}
     >{(msg: string) =>
-      <AnswerLabel aria-label={msg}>
-        {choiceIndicator}
+      <React.Fragment>
         <input
+          id={choiceIndicator}
```

Comment:
this seems like a pretty generic thing to make an id, is the id necessary for the label to work?

Reply:
if the input is outside of the label then it should be connected via `for` <-> `id` attributes

Reply:
yea but can you put it inside?

Reply:
In the behavior doc there is a screen showing a desired outline: https://docs.google.com/document/d/1gsfyJi5AdZSMUhbK6TFfzebXq6qU48sh4aD4MOo4EmY/edit#heading=h.hlfm2ekyfgmi

this outline is around the `label` and it should be visible when the user focuses the `input`
If `input` is inside the `label` then there is no way to style it that way

So currently user is focusing invisible `input` that is forcing the screenreader to read if the choice is selected or not and the visual indicator is around the label